build.sbt

```diff
@@ -129,17 +109,12 @@ lazy val examples = (project in file("examples"))
     (Compile / compile) := ((Compile / compile) dependsOn printSparkScalaVersion).value, // printSparkScalaVersion is run with compile
     mergeStrategy
   )
+  .settings(
+    jacocoReportSettings := commonJacocoReportSettings.withTitle("atum:examples Jacoco Report"),
+    jacocoExcludes := commonJacocoExcludes
+  )
```

Comment:
Examples should not have a Jacoco report. We are even skipping tests in them. They are not a deliverable package but rather a showcase.

Reply:
Code coverage measuring removed in commit 96d38265b64c3b092a624e733decebe52c716df9.